e2e-tests/web/test/specs/navigateToDashboard.e2e.ts

```diff
@@ -0,0 +1,34 @@
+import LandingPage from '../pageobjects/landing.page';
+
+describe('Landing Page', () => {
+    it('Filter and Navigate to City', async () => {
```

Comment:
```suggestion
describe('LandingPage', () => {
    it('should filter and navigate to city', async () => {
```

Reply:
Also I am not really sure if we shouldn't name the test case (`Landing Page`) the same way as the file name (`navigateToDashboard`) or the other way round :thinking:

Reply:
I think especially for the E2E tests it wouldn't make sense to use `Landin Page` as E2E tests should enclose UserStories, which include multiple use cases. But maybe a better naming would be sth. like `navigation`, `search`, ... which would couple multiple cases

Reply:
Yeah I agree. I just think file name should be the same as describe